assets/js/customize-preview.js

```diff
@@ -43,15 +43,6 @@
 				textColor = ! isDark ? 'var(--global--color-dark-gray)' : 'var(--global--color-light-gray)',
 				tableColor = ! isDark ? 'var(--global--color-light-gray)' : 'var(--global--color-dark-gray)';
 
-			// Modify the body class depending on whether this is a dark background or not.
```

Comment:
This class is used for other color settings, like changing the background of the input fields, so it needs to be kept.

Reply:
Added back